frontend/src/restapi/players-profiles/PlayersProfilesRestApi.ts

```diff
@@ -26,5 +27,11 @@ export const UserProfileRestApi = (
         .get<PlayerProfilesListDto>(`${currentConfig.baseUrl}/players-profiles`)
         .then((response) => response.data);
     },
+    async addPlayersProfile(playerProfile: PlayerProfileDto): Promise<void> {
```

Comment:
Byłbym tutaj za prefixowaniem wg. metody HTTP, że to jest **post**PlayersProfile

Reply:
ok, też o tym myślałem, ale zrobiłem jednak add bo lepiej brzmi :-p